site/docs/design_tokens/border-radius.mdx

```diff
@@ -0,0 +1,58 @@
+---
+name: Border Radius
+menu: Design tokens
+route: /design-tokens/border-radius
+---
+
+import LargeParagraph from "../../src/components/LargeParagraph";
+
+
+# Border Radius
+
+<LargeParagraph>The border-radius tokens are used to give elements rounded corners.</LargeParagraph>
+
+The border-radius CSS property rounds the corners of an element's outer edge. The radius applies to the background element, even if the element has no border.
+
+## Principles
+
+When creating components or editing existing ones, keep in mind, that rounded corners, whether small or big, **should never be applied to any part of the Helsinki brand assets**. 
+Border redius is only reserved for specific user interface needs. **Rounding should never be used as a mere stylization.** 
+
+## Usage
+
+### When to use
+
+- **To make a distinction between buttons and tags vs. keywords.**
+- **With elements that traditionally use rounding, E.g. media controls.**
+- **When emphasizing human communication, E.g. in chat speech bubbles.**
+- **Visual distinction from the surrounding elements is needed.** As humans, we are conditioned to favor rounded corners, while sharp corners invoke a feeling of danger.
+- **When designing for a very young demographic.** As children, we quickly learn that sharp corners hurt and that rounded corners are safer.
+- **Reproducing natural or some cultural connotations.** Rounded corners can be found everywhere in nature and humans have – throughout history – tried to replicate shapes found in nature.
+
+
+### When not to use
+
+- **Oversaturating your design with rounded corners isn’t a good idea**, unless you’re designing for a very young demographic. When to use rounded corners depends on the emotions that you want your users to experience.
```

Comment:
> Oversaturating your design with rounded corners isn’t a good idea, unless you’re designing for a very young demographic. When to use rounded corners depends on the emotions that you want your users to experience.

Vaikkakin ihan hyvä pointti, niin jättää aika paljon tulkinnanvaraa - erityisesti tuo viimeinen lause. Ottaen huomioon kuinka vähän näitä Helsingissä sallitaan, niin olisin näissä mieluummin liian tiukka.

Reply:
Samaa mieltä mutta ehkä aika iso homma käydä niitä kaikkia läpi?